FXyz-Samples/src/main/java/org/fxyz3d/samples/importers/ImportOBJ.java

```diff
@@ -62,9 +62,7 @@
     @Override
     protected void createMesh() {
         try {
-            model = (Group) Importer3D.load(
-                    ImportOBJ.class.getResource("DukeKing.obj").toExternalForm(),
-                    asPolygonMesh);
+            model = Importer3D.loadAsPoly(ImportOBJ.class.getResource("/org/fxyz3d/importers/DukeKing.obj")).getRoot();
```

Comment:
Why did you change the path? The obj file is not in that path. It is `/org/fxyz3d/samples/importers/DukeKing.obj`, so you can use `getResource("DukeKing.obj")`

Reply:
I don't believe I did. Not sure, maybe different old version?

Reply:
Samples are [here](https://github.com/FXyz/FXyz/tree/master/FXyz-Samples/src/main/resources/org/fxyz3d/samples/importers), don't you have this? Does the sampler work for you if you load these?

Reply:
There were probably changes to this after I forked. Will fix